### PR TITLE
[DOC] update scalars documentation & split API methods description to its own page

### PR DESCRIPTION
**Pull request by 36000**

### CODE REVIEW

**Comment by 36000** [comment]
@arokem this is some minor changes to the docs, should be ready for review / merge

File: docs/source/usage/scalars.rst [hunk 1]
```diff
@@ -1,7 +1,9 @@
-The pyAFQ Custom Scalars API
-~~~~~~~~~~~~~~~~~~
-pyAFQ has a system for users to add custom scalars (scalars beyond the several
-we calculate by default). The scalars API is similar to our Mask API.
+The pyAFQ Custom Structural Properties API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+pyAFQ has a system for users to add additional structural properties
```
Comment (arokem) [inline docs/source/usage/scalars.rst:3]:
```suggestion
Per default, pyAFQ calculates tract profiles of the FA and MD in every bundle that it recognizes. 
However, there is also a system for users to add tissue properties
```

File: docs/source/usage/scalars.rst [hunk 2]
```diff
@@ -1,7 +1,9 @@
-The pyAFQ Custom Scalars API
-~~~~~~~~~~~~~~~~~~
-pyAFQ has a system for users to add custom scalars (scalars beyond the several
-we calculate by default). The scalars API is similar to our Mask API.
+The pyAFQ Custom Structural Properties API
```
Comment (arokem) [inline docs/source/usage/scalars.rst:1]:
```suggestion
The pyAFQ Custom Tissue Properties API
```

File: docs/source/usage/scalars.rst [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+pyAFQ has a system for users to add additional structural properties
+or other additional images (beyond the several we calculate by default).
```
Comment (arokem) [inline docs/source/usage/scalars.rst:4]:
```suggestion
or other scalar images (i.e., 3D images with one numeric value per voxel).
```

**Review by arokem — COMMENTED** [review]
Just a few small suggestions.

**Pull request merged**